kitchensink/utils.py

```diff
@@ -15,9 +13,11 @@ def client_url(model, id=None, domain=None):
     A filter for template engine to generate URLs that open
     on fulfil client of the instance::
         <a href="{{ 'product.product'|client_url }}">Product List</a>
-        <a href="{{ 'product.product'|client_url(product.id) }}">{{ product.name }}</a>
+        <a href="{{ 'product.product'|client_url(product.id) }}">
+                 {{ product.name }}</a>
     A more sophisticated example with filter
-        <a href="{{ 'product.product'|client_url(domain=[('salable', '=', True)]) }}">
+        <a href="{{ 'product.product'
```

Comment:
Really man, I have to comment on this specifically don't you think it is quite obvious to apply same change everywhere?